reviews/2022-cert-manager-annual.md

```diff
@@ -0,0 +1,68 @@
+# cert-manager 2022 Annual Review
```

Comment:
general comment: from [Slack](https://kubernetes.slack.com/archives/CDEQJ0Q8M/p1639491869370100?thread_ts=1639491071.368500&cid=CDEQJ0Q8M), James Munnelly suggested:

> I wonder if we could/should break down the different other areas of the project (i.e. some of the CSI drivers that Josh has been working on, approval policy, csi-lib etc) to show that we’ve started to make a focus on extensions to the project too, and providing neutral APIs/standards for users to ‘plug in’ to the project

Reply:
another comment from [slack](https://kubernetes.slack.com/archives/CDEQJ0Q8M/p1639497310371100?thread_ts=1639491071.368500&cid=CDEQJ0Q8M):

> you can also mention the pca-issuer and the donation to cert-manager on it

Reply:
the above are now incorporated :+1: